index.html

```diff
@@ -0,0 +1,90 @@
+
+    <label for="number" id="number-label" name="age">Age</label>
+    <input type="number" class="input" id="number" placeholder="Age" min="18" max="35" required="required">
+    
+    <label for="dropdown">        Which option best describes your current role?
+    </label>
+    <select name="drowpdown" class="input" id="dropdown" name="select current role">
+        <option value="student">Student</option>
+        <option value="Fulltime-job">Fulltime-Job</option>
+        <option value="Fulltime-leaner"> Fulltime-leaner</option>
+    </select>
+
+    <p>Would you recommend freeCodeCamp to a friend?</p>
+        <input type="radio" value="radio" id="radio" name="radio" id="radio"> 
+        <label for="definitely">Definitely</label> <br>
+
+        <input type="radio"  value="radio" id="radio" name="radio" id="radio"> 
+        <label for="definitely">Maybe</label> <br>
+
+        <input type="radio"  value="radio" id="radio" name="radio" id="radio"> 
+        <label for="definitely">Not sure</label> <br> <br>
+
+    
+        <label for="dropdown">       What is your favorite feature of freeCodeCamp?
+        <select name="drowpdown" class="input" id="dropdown" name="select current role">
+            <option value="student">Challenges</option>
+            <option value="Fulltime-job">Project</option>
+            <option value="Fulltime-leaner"> Community</option>
+        </select>
+
+        <p>What would you like to see improved? (Check all that apply)</p>
+
+        <input type="checkbox" id="checkbox" name="checkbox" value="checkbox">
+        <label for="checkbox">Front-end Projects</label> <br>
+
+        <input type="checkbox" id="checkbox" name="checkbox" value="checkbox">
+        <label for="checkbox">Back-end Projects</label><br>
+
+        <input type="checkbox" id="checkbox" name="checkbox" value="checkbox">
+        <label for="checkbox">Data Visualization</label><br>
+
+        <input type="checkbox" id="checkbox" name="checkbox" value="checkbox">
+        <label for="checkbox">Challenges</label><br>
+
+        <input type="checkbox" id="checkbox" name="checkbox" value="checkbox">
+        <label for="checkbox">Open Sourse Community</label><br>
+
+        <input type="checkbox" id="checkbox" name="checkbox" value="checkbox">
+        <label for="checkbox">Gitter help rooms</label><br>
+
+        <input type="checkbox" id="checkbox" name="checkbox" value="checkbox">
+        <label for="checkbox">Videos</label><br>
+
+        <input type="checkbox" id="checkbox" name="checkbox" value="checkbox">
+        <label for="checkbox">City Meetups</label><br>
+
+        <input type="checkbox" id="checkbox" name="checkbox" value="checkbox">
+        <label for="checkbox">Wiki</label><br>
+
+        <p>Any comments or suggestions?</p>
+        <textarea name="comment" id="comment" cols="30" rows="10" placeholder="Enter your comment here..."></textarea>
+
+        <button type="submit" id="submit">Submit</button>
+
+</form>
```

Comment:
I think making the form section properly indented would add to the readability of your code.

Reply:
- [OPTIONAL] Same here kindly add some emojis to make your comments more human-friendly. :space_invader: :space_invader: